joomla/joomla-cms
# [com_cache] Normalize purge view

#### Summary of Changes

This simple PR is to normalize the com_cahe view with the other backend views. Just cosmetic changes.
##### Before PR

![image](https://cloud.githubusercontent.com/assets/9630530/14267763/569d456e-facb-11e5-8403-f788029b6c9a.png)
##### After PR

![image](https://cloud.githubusercontent.com/assets/9630530/14267760/5008cdcc-facb-11e5-8323-5da74626b5e2.png)
#### Testing Instructions
1. Use Joomla 3.5.0 or staging
2. Apply patch
3. Go to System -> Clear Expired cache
4. Check the visual
